### PR TITLE
Add support for Rocky Linux

### PR DESCRIPTION
**Pull request by estedev**
# Description

This allows the cookbook to be used on Rocky Linux.

If for any reason there'd be no appetite to support Rocky Linux in the sc-mongodb OSS cookbook, we'd be happy to "pivot" the approach and instead expose the ["package_repo_url"](https://github.com/sous-chefs/sc-mongodb/blob/main/recipes/mongodb_org_repo.rb#L30) as an attribute (default nil) and only get it filled out by the existing case logic if it's nil.
It'll keep the cookbook up and running for the currently supported OSs and we then set the attribute specifically for our Rocky purposes...

Thanks in advance for your feedback!

Kind regards,
David @ Escape Technology

### CODE REVIEW

**Comment by ramereth** [comment]
@estedev can you please update the changelog?

**Comment by estedev** [comment]
Sure, I've just updated CHANGELOG.md  to suggest a bump to v5.2.0 for the "Add support for Rocky Linux" PR.

Thanks in advance for your feedback!

Kind regards,
David @ Escape Technology

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 5.2.0 - *2023-05-26*
```
Comment (ramereth) [inline CHANGELOG.md:5]: Please remove this line, it'll automatically get updated by our system.

**Review by ramereth — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (estedev) [reply CHANGELOG.md]: Done!

**Pull request merged**